components/http_client.rst

```diff
@@ -0,0 +1,355 @@
+.. index::
+   single: HttpClient
+   single: Components; HttpClient
+
+The HttpClient Component
+========================
+
+    The HttpClient component is a low-level HTTP client with support for both
+    PHP stream wrappers and cURL. It also provides utilities to consume APIs.
+
+.. versionadded:: 4.3
+
+    The HttpClient component was introduced in Symfony 4.3.
+
+Installation
+------------
+
+.. code-block:: terminal
+
+    $ composer require symfony/http-client
+
+Alternatively, you can clone the `<https://github.com/symfony/http-client>`_
+repository.
+
+.. include:: /components/require_autoload.rst.inc
+
+Basic Usage
+-----------
+
+Use the :class:`Symfony\Component\HttpClient\HttpClient` class to create the
+low-level HTTP client that makes requests, like the following ``GET`` request::
+
+    use Symfony\Component\HttpClient\HttpClient;
+
+    $httpClient = HttpClient::create();
+    $response = $httpClient->request('GET', 'https://api.github.com/repos/symfony/symfony-docs');
+
+    $statusCode = $response->getStatusCode();
+    // $statusCode = 200
+    $contentType = $response->getHeaders()['content-type'][0];
+    // $contentType = 'application/json'
+    $content = $response->getContent();
+    // $content = '{"id":521583,"name":"symfony-docs",...}'
+
+If you are consuming APIs, you should use instead the
+:class:`Symfony\\Component\\HttpClient\\ApiClient` class, which includes
```

Comment:
Because here double backslashes are used

Reply:
Yes, inside `:class:` (and `:method:`, etc.) backslashes must be doubled to avoid syntax issues (it'd be nice to have this as an automated doc review rule).